extraConfigSriov.py

```diff
@@ -38,6 +40,33 @@ def _sno_make_deploy(
     client.oc("create namespace openshift-sriov-network-operator")
     client.oc("label ns --overwrite openshift-sriov-network-operator " "pod-security.kubernetes.io/enforce=privileged " "pod-security.kubernetes.io/enforce-version=v1.24 " "security.openshift.io/scc.podSecurityLabelSync=false")
 
+    ocp_version = None
+    v, number_of_subs_made = re.subn("^(4[.][0-9]+).*$", "\\1", version)
+    if number_of_subs_made == 1:
+        if v in ("4.14", "4.15", "4.16", "4.17"):
```

Comment:
We could also use `quay_tags = ("4.14", "4.15", "4.16", "4.17")` and use `quay_tags` for more readable code.